requirements/requirements_multimodal.txt

```diff
@@ -4,9 +4,9 @@ clip
 decord; sys_platform == 'linux'
 diffusers>=0.19.3
 einops_exts
+git+https://github.com/NVIDIA/Megatron-Energon.git
```

Comment:
don't think use main is a good idea, at least pin to one of commit?

Reply:
So I think energon folks will release a new version soonish (Within next 24 hrs) and then I will update this to that version. I agree we should not use main.